OptimizelySDK/Utils/Validator.cs

```diff
@@ -103,14 +103,24 @@ public static bool IsFeatureFlagValid(ProjectConfig projectConfig, FeatureFlag f
             return true;
         }
 
+        private static Type[] AttributeTypes =
```

Comment:
typeof generally takes too much time to get the type. Why we just don't replace with attribute types with string[].